src/browser/api_protocol/api_handlers/authorization.js

```diff
@@ -159,7 +160,7 @@ function addPendingAuthentication(uuid, token, file, sponsor, authReqPayload) {
 }
 
 function authenticateUuid(authObj, authRequest, cb) {
-    if (ExternalApplication.getExternalConnectionByUuid(authRequest.uuid) || coreState.getAppByUuid(authRequest.uuid)) {
+    if (ExternalApplication.getExternalConnectionByUuid(authRequest.uuid) || coreState.getAppByUuid(authRequest.uuid) || !(authRequest.runtimeClient || isUuidAvailable(authRequest.uuid))) {
```

Comment:
This is the way to go, all of our external adapts have to send `runtimeClient` right?

Reply:
No other adapters send this property, so it will be `undefined`.

(This is a funny flag, considering there was a pre-existing property on the auth request: `authRequest.client.type` which has values like `'dotnet'` or `'java'`. It might not hurt to start setting `authRequest.client.type = 'runtime'` for consistency. Too late to change it now.)